specification/digitaltwins/resource-manager/Microsoft.DigitalTwins/stable/2023-01-31/digitaltwins.json

```diff
@@ -2321,7 +2329,16 @@
       "in": "query",
       "description": "Specifies whether or not to attempt to clean up artifacts that were created in order to establish a connection to the time series database. This is a best-effort attempt that will fail if appropriate permissions are not in place. Setting this to 'true' does not delete any recorded data.",
       "required": false,
-      "type": "boolean",
+      "enum": [
+        "true",
```

Comment:
This is not a good practice to use 'true' and 'false' as enum values. Its very hard to add other meaningful enum values when you use booleans as enum values. Try to come up with another meaningful, extensible way to model what you want to configure. E.g. instead of "cleanupConnectionArtifacts is one of (true, false)" how about "cleanupArtifactsAutomatically" is one of (none, connections)"